src/modules/manage-team/ManageTeamPage.module.css

```diff
@@ -7,70 +7,83 @@
   gap: 2rem;
 }
 
+.container > div:nth-child(2) {
+  width: 100%;
+}
+
+.container > div:nth-child(1) {
+  height: min-content;
+}
+
 @media only screen and (max-width: 768px) {
   .container {
     flex-direction: column;
     gap: 0rem;
   }
 }
 
-.menu_segment {
-  padding: 2rem;
+.menu_card_content {
   display: flex;
   flex-direction: column;
   justify-items: center;
   gap: 1rem;
-  width: 206px;
+  width: 142px;
```

Comment:
Feels a bit like a magic value?  is this a multiplication of some base value? why did you end up on 142? and not 143 or 141?